js/renovation/ui/button.tsx

```diff
@@ -34,10 +34,10 @@ const getCssClasses = (model: ButtonProps): string => {
   const isValidStylingMode = stylingMode && stylingModes.includes(stylingMode);
   const classesMap = {
     'dx-button': true,
-    [`dx-button-mode-${isValidStylingMode ? stylingMode : 'contained'}`]: true,
+    [`dx-button-mode-${isValidStylingMode ?? false ? stylingMode : 'contained'}`]: true,
```

Comment:
let's move nullish coalescing to `isValidStylingMode` initialization

Reply:
ok